hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/insights/om/om.tsx

```diff
@@ -614,7 +614,7 @@ export class Om extends React.Component<Record<string, object>, IOmdbInsightsSta
     cancelDeletePendingSignal = controller;
 
     request.then(deletePendingKeysResponse => {
-      const deletePendingKeys = deletePendingKeysResponse && deletePendingKeysResponse.data && deletePendingKeysResponse.data.deletedKeyInfo;
+      const deletePendingKeys = deletePendingKeysResponse?.data?.deletedKeyInfo;
```

Comment:
Similar over here, please add a fallback/default value
```suggestion
      const deletePendingKeys = deletePendingKeysResponse?.data?.deletedKeyInfo ?? [];
```